iwyu_output.cc

```diff
@@ -468,8 +468,8 @@ bool OneIncludeOrForwardDeclareLine::HasSymbolUse(const string& symbol_name)
   return ContainsKey(symbol_counts_, symbol_name);
 }
 
-void OneIncludeOrForwardDeclareLine::AddSymbolUse(const string& symbol_name) {
-  ++symbol_counts_[symbol_name];
+void OneIncludeOrForwardDeclareLine::AddSymbolUse(const string& symbol_name, int multiplicity) {
```

Comment:
Can you remove this method to `AddSymbolsUses` and call `multiplicity` `count` instead?